content/concept-xrp-ledger-intro.md

```diff
@@ -0,0 +1,88 @@
+- Each account in the XRP Ledger must hold a small amount of XRP in reserve. This is an anti-spam measure to disincentivize making the ledger data occupy too much space. XRP Ledger validators can vote to change the amount of XRP required as a reserve, to compensate for changes in XRP's real-world value. (The last time this happened was in December 2013, when [the reserve requirement decreased from 50 XRP to 20 XRP](https://ripple.com/insights/proposed-change-to-ripple-reserve-requirement-2/).) If the reserve requirement decreases, XRP that was previously locked up by the reserve becomes available again.
+- Ripple (the company) holds a large reserve of XRP in escrow. At the start of each month, 1 billion XRP is released from escrow for Ripple to use. (Ripple uses XRP to incentivize growth in the XRP Ledger ecosystem and sells XRP to institutional investors.) At the end of each month, any remaining XRP the company does not sell or give away is stored into escrow for a 54-month period. For more information on Ripple's escrow policy, see [Ripple Escrows 55 Billion XRP for Supply Predictability](https://ripple.com/insights/ripple-to-place-55-billion-xrp-in-escrow-to-ensure-certainty-into-total-xrp-supply/). For more information on the technical capabilities of the Escrow feature, see [Escrow](concept-escrow.html).
+
+
+## Responsible Software Governance
+[Responsible Software Governance]: #responsible-software-governance
+
+Any piece of software can only be as good as the developers who code and manage it. Ripple employs a team of world-class engineers dedicated full-time to maintaining and improving the XRP Ledger software, especially the core server, `rippled`. The [source code for `rippled`](https://github.com/ripple/rippled/) is available to the public with a permissive open-source license, as are many other parts of the XRP Ledger ecosystem. Ripple engineers follow best practices for software engineering, including:
+
+- A famously strict and thorough code review process
+- Comprehensive code coverage and unit tests
+- Regularly running automated checks for potential vulnerabilities and memory leaks
+- Regularly commissioning external reviews by professional organizations
+
+The XRP Ledger also has a built-in "Amendments" system for evolving the existing feature set and transitioning from old rules to new ones. Rather than fracturing the ecosystem or causing uncertainty whenever a transition occurs, any changes to transaction processing in the XRP Ledger are switched on by Amendments that require sustained approval from validators over 2 weeks, giving everyone time to evaluate the changes before they apply, and upgrade their servers appropriately. The system also prevents outdated servers from reporting incorrect data when the rules of the network have changed. For more on how Amendments work, see [Amendments](concept-amendments.html).
```

Comment:
Consider breaking this out as its own section. Although it does fall under your `Responsible Software Governance` category, Amendments seem a bit different than the software engineering qualities listed above.